R/yaml_utils.R

```diff
@@ -238,3 +238,30 @@ as_yaml_auto <- function(input_list,
 print.rmd_yaml_header <- function(x, ...) {
   cat(x, ...)
 }
+
+#' Extract a Field from `yaml` and Optionally Retrieve Names from a List
+#'
+#' This function parses a `YAML` text and extracts the specified field. It provides
+#' the option to retrieve the names of elements from a list if the field contains a list.
```

Comment:
inaccurate

Reply:
Sorry about this. The description is outdated. Initially, there was an additional parameter to retrieve if the field is a list. I have updated the description.